package.json

```diff
@@ -11,8 +11,9 @@
     }
   },
   "scripts": {
-    "bootstrap": "lerna bootstrap --ignore-scripts && lerna exec npm i --scope=ethereumjs-block",
```

Comment:
`lerna bootstrap` now installs packages for ethereumjs-block as well, so removing `npm i` here.